debian/ocf-atool.postinst

```diff
@@ -0,0 +1,18 @@
+# create conf directory
+if [ ! -e /etc/ocf-atool ]; then
+	mkdir /etc/ocf-atool || { echo "Couldn't mkdir /etc/ocf-atool, exiting."; exit 1; }
+fi
+
+# create secret file
+if [ ! -e /etc/ocf-atool/secret ]; then
+	echo "No Django secret exists, generating one automatically..."
+
+	tmp=$(mktemp --tmpdir=/etc/ocf-atool)
+	pwgen -s 64 -n1 > "$tmp"
+	chown atool:root "$tmp"
+	chmod 400 "$tmp"
+
+	mv "$tmp" /etc/ocf-atool/secret || { echo "Couldn't write to /etc/ocf-atool/secret, exiting."; exit 1; }
```

Comment:
it doesn't really matter here anyway, but i'm more concerned about the malicious case (e.g. an evil user makes a bad symlink which we then write into) than clobbering a file.

`mv -n` is still vulnerable to race conditions afaik (at least based on strace output), so I don't think yours improves anything. and not sure what we get from running chown/chmod before pwgen?